lib/parse.js

```diff
@@ -187,6 +188,10 @@ module.exports = function (str, opts) {
         throw new TypeError('Decoder has to be a function.');
     }
 
+    if (options.strictNullHandling === true && has.call(options, 'emptyValue')) {
```

Comment:
we check truthiness of `strictNullHandling` elsewhere; why `=== true` here?

Reply:
I think the previous conversation was lost on rebase, we had said that it doesn't really make sense for `strictNullHandling` and `emptyValue` to be used together, since both try to determine what the "empty" value is.

Reply:
oh sure, i meant why not this:

```suggestion
    if (options.strictNullHandling && has.call(options, 'emptyValue')) {
```

Reply:
Ah, gotcha. Thanks for explaining!